Library/Formula/kapacitor.rb

```diff
@@ -16,26 +16,23 @@ class Kapacitor < Formula
     sha256 "f948ff298c8bdac82b0caefb743a834348d67b38644b2746dea6ae720cca4275" => :mavericks
   end
 
+  option "with-foo", "Compile with foo bindings"
```

Comment:
Possibly a documentation copy/paste that was unused; It's the same wording as in our Cookbook.